modify-word/meta.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Modify Word",
+  "name": "modify_word",
+  "difficulty": "hard",
+  "author": "Tykhon",
+  "category": "logic"
+}
```

Comment:
3 suggestions:
1- Write your full name in the "author" line.
2- I would suggest the category "string-3" instead of "logic", since it mainly takes in a string, and its difficulty is hard.
3- In line 3 of the io.json, there's a mistake. It's supposed to be written, "input": ["Hello"], and not "input": ["Hoell"],
Everything else looks fine!

Reply:
Ok,thanks.